.github/workflows/test.yml

```diff
@@ -23,13 +23,12 @@ jobs:
         run: |
           bash ./.github/scripts/install_dependencies.sh
 
-      - name: Build and test
+      - name: Build
         run: |
           mkdir -p build
           cd build
-          cmake -DBUILD_TESTS_EXAMPLES=ON ..
```

Comment:
You probably still want this so it actually tests linux. No binary is packaged here so it fine if you want to avoid updating the license

Reply:
Ok.